frontend/src/metabase/components/Popover/TippyPopover.tsx

```diff
@@ -126,15 +123,11 @@ function TippyPopover({
     [flip, sizeToFit, popperOptions],
   );
 
-  const zIndex =
-    theme.other.popover?.zIndex ||
-    ("var(--mb-overlay-z-index)" as unknown as number);
```

Comment:
We don't need `theme.other.popover.zIndex` anymore thanks to the CSS variable definition generation. This is a cleanup so we rely on CSS variables instead of direct Mantine theme object access.